errors.py

```diff
@@ -38,7 +38,7 @@ class Redirect( HTTPException ):
 		Causes an 303 - See Other (or 302 - Found if requested / 301 - Moved Permanently) redirect
 	"""
 	def __init__( self, url, descr="Redirect", status=303 ):
-		super( Redirect, self ).__init__(  status=303, name = "Redirect", descr=descr )
+		super( Redirect, self ).__init__(  status=status, name="Redirect", descr=descr )
```

Comment:
you might check for approriate status codes here, so nobody is a ble to send status 200 or something else via errors.Redirect()

```python
if status not in [301, 302, 303]: ..
```
I'm not sure if its better to let status default to 301 in this case or to raise an exception out of this exception...